scripts/readme/ghactions_driver/readme_workflow_generate.py

```diff
@@ -46,7 +46,10 @@ def write_readme_workflow(readme_path, output_telemetry=Telemetry()):
     else:
         ReadmeSteps.extract_steps_and_run()
 
-    ReadmeStepsManage.write_workflow(
-        workflow_name, "samples_readme_ci", output_telemetry
-    )
+    if workflow_name.endswith("pdf"):
```

Comment:
we have anyother flow to treat otherwise "chat_with_pdf"? If not, why specify the flow name instead. Also please add some comments to document why we need this special logic.

Reply:
OK, I'll add some comments. add 3.11 to chat with pdf is bug founded from Lisa, even though we finally find out it is not  a python version problem, but it also reveals some potential issues. Just enable multiple chat-with-pdf since it is running long and easy to find problems.